### DISCUSSION TITLE
Still active?

### DISCUSSION THREAD

**Discussion by ArtMattner** [Yakifo/amqtt#162] [General]
Hello,
just asking myself if this project is still active?
Because the latest announcement was on the beta-release, I'm wondering which run now stable on modern Python versions?
Can't really test it on Python3.7 because apple silicon doesn't support that old version <3.8
Latest version 0.10.1 doesn't run proper on Python3.12.x
Which Python version you can suggest?

Another question targets the question about MicroPython. Is there any activity on that one?

Thanks and BR,
Art

**Comment by HerrMuellerluedenscheid** [comment]
@ArtMattner thanks for raising the topic!

I personally am super busy and shifted focus. Therefore I would be very much in favor seeking for more maintainers. @FlorianLudwig what do you think? I have the impression that you are equally overbooked with work.

**Reply by ArtMattner** [reply]
No worries, I'm busy myself.
Soon or later I will get into the nice situation of vacation, then I will try first something on my machine.
If I find something that is worth to take a look onto, I will drop a message here so you can at least have a look into. ;-)

**Comment by ajmirsky** [comment]
@ArtMattner I have recently joined the maintainers squad for this project and am moving swiftly to get a new version released. Key changes in this release:

- support for python 3.10+
- upgrade to websocket package
- updated documentation, specifically with more detail on broker configuration

There have been a few pull requests opened thus far and any feedback from the community is appreciated!

**Comment by ajmirsky** [comment]
@ArtMattner roadmap for the project can be found within [milestones](https://github.com/Yakifo/amqtt/milestones)
